lib/Feature/FeatureModelTransaction.cpp

```diff
@@ -36,6 +40,86 @@ void removeFeature(FeatureModel &FM,
   Trans.commit();
 }
 
+bool fvIsLeave(FeatureModel &FM, detail::FeatureVariantTy &FV) {
+  Feature *ActualFeature;
+  std::visit(
+      Overloaded{[&ActualFeature, &FM](Feature *F) { ActualFeature = F; },
+                 [&ActualFeature, &FM](string &F) {
+                   ActualFeature = FM.getFeature(F);
+                 }},
+      FV);
+  return ActualFeature->isLeave();
```

Comment:
What happens here if the FM does not have a feature with the given name?

Reply:
Done --> Such elements won't be deleted.